charts/telepresence/values.yaml

```diff
@@ -136,6 +136,25 @@ agentInjector:
     name: mutator-webhook-tls
   certificate:
     regenerate: false
+
+    # The method used to generate the TLS certificate for the agent-injector.
+    #
+    # Possible options: helm ou certmanager
+    #
+    # Default: helm
+    method: helm
+
+    # The certmanager configuration block
+    #
+    # Default: {}
+    certmanager: {}
```

Comment:
Please add a brief description for these new values to the [charts/telepresence/README.md](https://github.com/telepresenceio/telepresence/blob/release/v2/charts/telepresence/README.md).

Reply:
I will, thx!